OneCommunityGlobal/HighestGoodNetworkApp
# NEEDS NEW DEVELOPER - Create the ability to approve suggestions

# Description
Fix MANAGEMENT DASHBOARD COMPONENT
(PRIORITY HIGH) Yiyun: Create the ability for admin/owner users to approve and apply “suggested” changes on a task.

## Mainly changes explained:
Add some functions to make the Suggest button work. Also, the suggested changes list should be removed from the dashboard.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log in as mentor/manager role
4. go to dashboard→ Tasks→ click on the tasks near the red bell→ click on "Suggest" and make some changes.
5. log out the current user
6. log in as admin/owner role
7. click on the red circle(notification) near "Dashboard"
<img width="183" alt="Screen Shot 2023-03-18 at 3 32 43 PM" src="https://user-images.githubusercontent.com/70128933/226137078-528b91bb-7115-4028-bba3-ce997a69bd35.png">
→ click on the task suggestion you made using the former role→ Approve→ Go back to the former user and click on the red bells to see whether the changes have been made already.

## Note:
There are many log in log out in this PR, try to use different browsers for different role, and refresh the page often.
If you do not have team member tasks, login in as admin, go to dashboard→ Other Links→ Teams→ create your own team to test.
